dev/breeze/src/airflow_breeze/prepare_providers/provider_documentation.py

```diff
@@ -1208,8 +1215,6 @@ def update_changelog(
             maybe_with_new_features=maybe_with_new_features,
             with_min_airflow_version_bump=with_min_airflow_version_bump,
         )
-    get_console().print(f"\n[info]Update index.rst for {package_id}\n")
```

Comment:
Moved to sepearate method that is always called - even if changelog generation is skipped.